.cron.yml

```diff
@@ -4,15 +4,6 @@
 ---
 
 jobs:
-    - name: snapshot
-      job:
-          type: decision-task
-          treeherder-symbol: snapshot-D
-          target-tasks-method: snapshot
```

Comment:
Do we want to remove the `snapshot` target task too? Or do you want to handle this in a later step?

Reply:
For now I just want to reduce the frequency to zero to see who complains. @pocmo and I aren't sure if they are potentially other downstream consumers that we don't know of so we wanted to take this gradually. 

In the meantime, CloudOps is looking into logs/stats to see if we're still getting GET requests to these repos. My current plan is to wait for another week:
a) if nobody complains in the meantime for stalling snapshots
b) CloudOps confirms we no longer have incoming trafic to snapshots maven repo

... then I'll follow-up with removing all the moving pieces from both this repository, beetmover, ciadmin and such.